<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>413</width>
    <height>80</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>80</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>80</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#Form{
	background-color: transparent;
	border: none;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <widget class="QFrame" name="framewid">
     <property name="styleSheet">
      <string notr="true">#framewid{
	background-color: rgb(250, 250, 250);
	border: 0px solid  rgb(250, 250, 250);
	border-radius: 5px;
}
</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>4</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>4</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QFrame" name="frame_2">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: transparent;
</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QLabel" name="labelExt">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>4</x>
           <y>3</y>
           <width>70</width>
           <height>70</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>70</width>
           <height>70</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>70</width>
           <height>70</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel{
	color: rgb(120, 120, 120);
	border: 2px solid  rgb(243, 116, 53);
	border-radius: 35px;
	font: 8pt url(:/plugins/StaraMaps/Roboto-Regular.ttf);
}
QLabel:disabled{
	border: 2px solid  rgba(100, 100, 100, 40);
}
</string>
         </property>
         <property name="text">
          <string>.shp</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_3">
        <property name="font">
         <font>
          <pointsize>5</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">#frame_3{
	background-color:transparent;
	border-bottom: 2px solid;
	border-bottom-color: rgb(235, 235, 235);
}

</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>5</number>
         </property>
         <property name="leftMargin">
          <number>5</number>
         </property>
         <property name="topMargin">
          <number>5</number>
         </property>
         <property name="rightMargin">
          <number>5</number>
         </property>
         <property name="bottomMargin">
          <number>5</number>
         </property>
         <item>
          <widget class="QLabel" name="labelFileName">
           <property name="font">
            <font>
             <family>:/plugins/StaraMaps/Roboto-Regular.ttf</family>
             <pointsize>12</pointsize>
             <weight>50</weight>
             <italic>false</italic>
             <bold>false</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(120, 120, 120);
font: 12pt url(:/plugins/StaraMaps/Roboto-Regular.ttf);</string>
           </property>
           <property name="text">
            <string>Filename</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelFileSize">
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>0</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>:/plugins/StaraMaps/Roboto-Regular.ttf</family>
             <pointsize>8</pointsize>
             <weight>50</weight>
             <italic>false</italic>
             <bold>false</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(180, 180, 180);
font: 8pt url(:/plugins/StaraMaps/Roboto-Regular.ttf);</string>
           </property>
           <property name="text">
            <string>400kB</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <property name="spacing">
            <number>5</number>
           </property>
           <item>
            <widget class="QComboBox" name="valueComboBox">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="minimumSize">
              <size>
               <width>150</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="layoutDirection">
              <enum>Qt::RightToLeft</enum>
             </property>
             <property name="autoFillBackground">
              <bool>false</bool>
             </property>
             <property name="styleSheet">
              <string notr="true">QComboBox{
	border:1px solid rgb(240, 240, 240);
	border-radius: 3px;
	background-color: rgb(250, 250, 250);
	color: rgb(120, 120, 120);
}

QComboBox::drop-down{
	border: 0px
}

QComboBox::down-arrow{
	border:none;
	width: 20px;
	height: 20px;
	image: url(:/plugins/StaraMaps/down.png);
	padding-left: 5px;
}
QComboBox::down-arrow:disabled{
	image: url(:/plugins/StaraMaps/down_disabled.png);
}

QComboBox:disabled{
	color: rgb(240, 240, 240);
}

QComboBox QAbstractItemView {
      border: 2px solid darkgray;
      selection-background-color: red;
  }
</string>
             </property>
             <property name="editable">
              <bool>false</bool>
             </property>
             <property name="currentText">
              <string>Test</string>
             </property>
             <item>
              <property name="text">
               <string>Test</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>test1</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QCheckBox" name="checkBox">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_del">
             <property name="minimumSize">
              <size>
               <width>30</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>30</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton{
	border:  1px solid rgb(243, 116, 53);	
	color:  rgb(243, 116, 53);
	border-radius: 5px;
	font: 12pt url(:/plugins/StaraMaps/Roboto-Regular.ttf);
	image: url(:/plugins/StaraMaps/excluir.png);
}

QPushButton::hover{
	/*background-color: rgb(233, 106, 43);*/
	border: 2px solid rgb(243, 116, 53);	
	color: rgb(250, 250, 250);
}

QPushButton::pressed{
	/*background-color: rgb(223, 96, 33);*/
	color: rgb(250, 250, 250);
}</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_up">
             <property name="minimumSize">
              <size>
               <width>40</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>40</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton{
	border:  1px solid rgb(243, 116, 53);	
	color:  rgb(243, 116, 53);
	border-radius: 5px;
	font: 12pt url(:/plugins/StaraMaps/Roboto-Regular.ttf);
	image: url(:/plugins/StaraMaps/setas-para-cima.png);
}

QPushButton::hover{
	background-color: rgb(233, 106, 43);
	color: rgb(250, 250, 250);
}

QPushButton::pressed{
	background-color: rgb(223, 96, 33);
	color: rgb(250, 250, 250);
}</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_4">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QPushButton" name="pushButtonOptions">
         <property name="geometry">
          <rect>
           <x>9</x>
           <y>26</y>
           <width>32</width>
           <height>32</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
	image: url(:/plugins/StaraMaps/options.png);
	padding: 2px;
	color: rgb(250, 250, 250);
	border: none;
}

QPushButton::hover{
	background-color: rgb(245, 245, 245);
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
